themes/agnoster.zsh-theme

```diff
@@ -79,8 +79,10 @@ prompt_end() {
 
 # Context: user@hostname (who am I and where am I)
 prompt_context() {
-  if [[ "$USER" != "$DEFAULT_USER" || -n "$SSH_CLIENT" ]]; then
+	if [[ "$USER" != "$DEFAULT_USER" ]]; then
```

Comment:
Indent is broken.
Could you change indents from 1 tab to 2 spaces?
FYI: https://github.com/robbyrussell/oh-my-zsh/wiki/Coding-style-guide#indentation